provableAPI_0.6.sol

```diff
@@ -0,0 +1,1326 @@
+            OAR = OracleAddrResolverI(0xc03A2615D5efaf5F49F60B7BB6583eaec212fdf1);
+            provable_setNetworkName("eth_ropsten3");
+            return true;
+        }
+        if (getCodeSize(0xB7A07BcF2Ba2f2703b24C0691b5278999C59AC7e) > 0) { //kovan testnet
+            OAR = OracleAddrResolverI(0xB7A07BcF2Ba2f2703b24C0691b5278999C59AC7e);
+            provable_setNetworkName("eth_kovan");
+            return true;
+        }
+        if (getCodeSize(0x146500cfd35B22E4A392Fe0aDc06De1a1368Ed48) > 0) { //rinkeby testnet
+            OAR = OracleAddrResolverI(0x146500cfd35B22E4A392Fe0aDc06De1a1368Ed48);
+            provable_setNetworkName("eth_rinkeby");
+            return true;
+        }
+        if (getCodeSize(0xa2998EFD205FB9D4B4963aFb70778D6354ad3A41) > 0) { //goerli testnet
+            OAR = OracleAddrResolverI(0xa2998EFD205FB9D4B4963aFb70778D6354ad3A41);
+            provable_setNetworkName("eth_goerli");
+            return true;
+        }
+        if (getCodeSize(0x6f485C8BF6fc43eA212E93BBF8ce046C7f1cb475) > 0) { //ethereum-bridge
+            OAR = OracleAddrResolverI(0x6f485C8BF6fc43eA212E93BBF8ce046C7f1cb475);
+            return true;
+        }
+        if (getCodeSize(0x20e12A1F859B3FeaE5Fb2A0A32C18F5a65555bBF) > 0) { //ether.camp ide
+            OAR = OracleAddrResolverI(0x20e12A1F859B3FeaE5Fb2A0A32C18F5a65555bBF);
+            return true;
+        }
+        if (getCodeSize(0x51efaF4c8B3C9AfBD5aB9F4bbC82784Ab6ef8fAA) > 0) { //browser-solidity
+            OAR = OracleAddrResolverI(0x51efaF4c8B3C9AfBD5aB9F4bbC82784Ab6ef8fAA);
+            return true;
+        }
+        return false;
+    }
+    /**
+     * @dev The following `__callback` functions are just placeholders ideally
+     *      meant to be defined in child contract when proofs are used.
+     *      The function bodies simply silence compiler warnings.
+     */
+    function __callback(bytes32 _myid, string memory _result) virtual public {
+        __callback(_myid, _result, new bytes(0));
+    }
+
+    function __callback(bytes32 _myid, string memory _result, bytes memory _proof) virtual public {
+      _myid; _result; _proof;
+      provable_randomDS_args[bytes32(0)] = bytes32(0);
+    }
+
+    function provable_getPrice(string memory _datasource) provableAPI internal returns (uint _queryPrice) {
+        return provable.getPrice(_datasource);
+    }
+
+    function provable_getPrice(string memory _datasource, uint _gasLimit) provableAPI internal returns (uint _queryPrice) {
+        return provable.getPrice(_datasource, _gasLimit);
+    }
+
+    function provable_query(string memory _datasource, string memory _arg) provableAPI internal returns (bytes32 _id) {
+        uint price = provable.getPrice(_datasource);
+        if (price > 1 ether + tx.gasprice * 200000) {
+            return 0; // Unexpectedly high price
+        }
+        return provable.query.value(price)(0, _datasource, _arg);
```

Comment:
.value(...) is deprecated as of 0.6.4. Might be nice to change this to {value: ...} instead.  Same deprecated usage in 15 other places below.